### ISSUE TITLE
Show tag value as column in Grafana Trace View widget

### ISSUE THREAD

**Issue by MurzNN** [grafana/tempo#1118]
**Is your feature request related to a problem? Please describe.**
At now in Trace View widget we can see list of spans and only duration value of each. But very often developers adds useful info to span tags. 

For example, very often developers adds `memory_usage` or `sql_queries_count` tags to each span.

Now we can see tags and their values, but to see them - we need to click on each span to expand them, and even with this we can't simply visually compare values of tags between different spans as in table list.

**Describe the solution you'd like**
Good solution for this problem can be ability to add new column to spans table, that displays value of desired tag for each span.

**Comment by joe-elliott** [comment]
@connorlindsey Heads up on this request. It would require changes on both Grafana and Tempo side, but I really like the idea.

**Comment by connorlindsey** [comment]
@joe-elliott I like this idea as well and can see how it'd be useful to have. It'll require some UX work and exploration, so I'll bring it up with the team.

**Comment by bentqc** [comment]
to add, it would be very practical to set something like error tag to quickly drill down on the traces. i get that we can search and list all traces using error=true, but what if you want to get all traces (including the ones that does not errored) and represent them both together in the same table view. Is adding the ability of altering the explore table to add custom tags considered today ?

**Comment by joe-elliott** [comment]
@connorlindsey can we move this to the `grafana/grafana` repo and prioritize it there?

**Comment by connorlindsey** [comment]
Yeah, I think moving it to a discussion in grafana/grafana would be best cc @joey-grafana

**Comment by bentqc** [comment]
Has there been any progress on this item please ?  @joe-elliott @connorlindsey @joey-grafana
